app/models/episode.rb

```diff
@@ -23,10 +23,10 @@ class Episode < ApplicationRecord
   serialize :overrides, HashSerializer
 
   belongs_to :podcast, -> { with_deleted }, touch: true
-  has_many :contents, -> { order("position ASC, created_at DESC") }, autosave: true, dependent: :destroy
+  has_many :contents, -> { order("position ASC, created_at DESC") }, autosave: true, dependent: :destroy, inverse_of: :episode
```

Comment:
Yeah, the advice I see is that the Rails built in ability to figure out these relationships is not great, if you set any other options, and there seems to be no harm in setting explicitly.